config/src/projects/zkswapv2.ts

```diff
@@ -0,0 +1,308 @@
+            name: 'ZkSyncCommitBlock',
+            upgradable: false,
+            owner: {
+              address: '0x9D7397204F32e0Ee919Ea3475630cdf131086255',
+              type: 'other',
+            },
+          },
+          {
+            address: '0x94b9401945a9bc06CE5B69e6dB3c6B671aABc829',
+            name: 'Verifier',
+            upgradable: false,
+            owner: {
+              address: '0x9D7397204F32e0Ee919Ea3475630cdf131086255',
+              type: 'other',
+            },
+          },
+          {
+            address: '0x95269f9E76540459c797089034dc74b48dF780a2',
+            name: 'Governance',
+            upgradable: false,
+            owner: {
+              address: '0x9D7397204F32e0Ee919Ea3475630cdf131086255',
+              type: 'other',
+            },
+          },
+        ],
+        risks: [
+          {
+            category: 'Funds can be stolen if',
+            text: 'the contract owner pushes a malicious code upgrade which takes 7 days.',
+          },
+        ],
+      },
+      operator: {
+        name: 'The validator is centralized.',
+        description:
+          'In the beginning ZKSwap is asking users to trust its centralized sequencer. Will be decentraliazed at later stage.',
+        risks: [
+          {
+            category: 'Funds can be lost if',
+            text: 'the sequencer exploits their centralized position and frontruns user transactions.',
+          },
+        ],
+        references: [
+          {
+            text: 'How validity proof works',
+            href: 'https://medium.com/starkware/validity-proofs-vs-fraud-proofs-4ef8b4d3d87a',
+          },
+        ],
+      },
+      forceTransactions: {
+        name: 'Users can force submit any withdraw transaction',
+        description:
+          'Because the state of ZKSwap is based on transactions submitted to the layer1 smart contract and anyone can submit their withdraw transactions there it allows the users to circumvent censorship by interacting with the smart contract directly.',
+        risks: [],
+        references: [],
+      },
+    },
+    purpose: 'Payments, Exchange',
+    riskView: {
+      stateCorrectness: RISK.SNARK_PROOFS,
+      dataAvailability: RISK.DATA_ON_CHAIN,
+      censorshipResistance: RISK.UNKNOWN,
+      upgradeability: RISK.UPGRADE_DELAY('2 weeks'),
+      owner: RISK.MULTISIG_OWNER,
+    },
+    parameters: [
+      {
+        name: 'Primary use case',
+        value: 'Payments, Exchange',
+      },
+      {
+        name: 'Hypothetical level of decentralization',
+        sentiment: 'good',
+        value: 'High',
+      },
+      {
+        name: 'Current level of decentralization',
+        tooltip: 'Contracts are upgradable',
+        sentiment: 'bad',
+        value: 'Low',
+      },
+      {
+        name: 'Can funds be stolen by the operator?',
+        tooltip:
+          'Contracts are upgradable BUT on the upside there is a 14 days timelock',
```

Comment:
When describing the contracts you specified the timelock as 7 days. Is there a way for someone to verify this?

Reply:
This is a mistake. It's 8 days in fact.